### PR TITLE
default action non empty check

### PR DESCRIPTION
**Pull request by pulkit8**
Check for non empty default action to fix structured message generic and list issue

### CODE REVIEW

File: Messages/MessageElement.php [hunk 1]
```diff
@@ -77,13 +79,23 @@ public function __construct($title, $subtitle, $image_url = '', $buttons = [], $
...
+                'default_action' => $this->default_action
+            ];
+        } else {
+            $result = [
+                'title' => $this->title,
+                'subtitle' => $this->subtitle,
+                'item_url' => $this->url,
+                'image_url' => $this->image_url,
+            ];
+        }
```
Comment (ghost) [inline Messages/MessageElement.php:98]:
```php
$result = [
    'title' => $this->title,
    'subtitle' => $this->subtitle,
    'item_url' => $this->url,
    'image_url' => $this->image_url,
];

if (!empty($default_action)) {
    $result['default_action'] = $this->default_action;
}
```

**Comment by wittfabian** [comment]
@pulkit8 Check the comment!

**Comment by yourchoice** [comment]
Take a look here: https://github.com/pimax/fb-messenger-php/issues/147#issuecomment-399897291

Maybe is better to init from __construct with null

File: Messages/MessageElement.php (hunk 1 above)
Comment (jason-engage) [reply Messages/MessageElement.php:98]: Wolzogens code is cleaner. Any pull request for this?

**Comment by pulkit8** [comment]
Another pull request merged for this file (#150 ). Closing this duplicate.

**Pull request closed without merging**